src/components/Modals/NewCourse/CourseSelector.vue

```diff
@@ -56,12 +56,14 @@ const getMatchingCourses = (
   return code.concat(title).slice(0, 10);
 };
 
-export default Vue.extend({
+export default defineComponent({
   props: {
     searchBoxClassName: { type: String, required: true },
     placeholder: { type: String, required: true },
     courseFilter: {
-      type: Function as PropType<((course: CornellCourseRosterCourse) => boolean) | undefined>,
+      type: (Function as unknown) as PropType<
```

Comment:
Unknown?

Reply:
Yes... not sure why there is this error if I don't cast to unknown first